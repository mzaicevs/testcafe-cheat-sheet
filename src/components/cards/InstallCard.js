import React from "react";
import { install } from "../../snippets";
import CodeCard from "../../base_components/CodeCard";

const InstallCard = () => (
  <CodeCard
    cardTitle={install.title}
    codeSnippet={{
      code: install.code,
      language: install.language
    }}
  />
);

export default InstallCard;
