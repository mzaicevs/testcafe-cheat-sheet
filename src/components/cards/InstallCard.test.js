import React from "react";
import { shallow } from "enzyme";
import InstallCard from "./InstallCard";

jest.mock("react-syntax-highlighter/dist/esm/styles/hljs", () => () => {});

describe("InstallCard", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<InstallCard />);
    expect(wrapper).toMatchSnapshot();
  });
});
