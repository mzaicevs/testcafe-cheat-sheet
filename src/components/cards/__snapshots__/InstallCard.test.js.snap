// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstallCard matches snapshot 1`] = `
<CodeCard
  cardTitle="Install TestCafe"
  codeSnippet={
    Object {
      "code": Array [
        "npm install -D testcafe",
      ],
      "language": "plaintext",
    }
  }
/>
`;
